web/vtadmin/.gitignore

```diff
@@ -23,3 +23,7 @@ yarn-debug.log*
 yarn-error.log*
 
 .eslintcache
+
+# @planetscale/vtadmin
```

Comment:
These are only ever used by Planetscale to release the @planetscale/vtadmin npm package.